snap7/common.py

```diff
@@ -44,6 +46,14 @@ def __new__(cls, *args, **kwargs):
         return cls._instance
 
     def __init__(self, lib_location: Optional[str] = None):
+        """ Loads the snap7 library using ctypes cdll.
+
+        Args:
+            lib_location: full path to the `snap7.dll` file. Defaults to None
```

Comment:
From the type information, one already knows the default value is none. I would add "optional".